src/app_service/common/activity_center.nim

```diff
@@ -0,0 +1,8 @@
+import json
+
+import ../../app/core/eventemitter
+
+const SIGNAL_RAW_ACTIVITY_CENTER_NOTIFICATIONS* = "rawActivityCenterNotifications"
```

Comment:
Would be nice to add a verb to the signal name. But it seems to be used in so many different cases that it might be hard to find a good suitable single verb 😄